composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": "^5.3.2|^7.0",
-        "symfony/framework-bundle": "~2.0|^3.0",
+        "symfony/framework-bundle": "~2.0|^3.1",
```

Comment:
Oups, so sorry indeed, but I can not to install your bundle when I tried, it returned me the next message :
 "- Conclusion: remove symfony/symfony v3.1.5
    - Conclusion: don't install symfony/symfony v3.1.5".
Can you help me ? 
thanks